Automattic/sensei
# Skip opinionated styles when the active theme declares support for it

Resolves #5982  #5983 
### Introduced changes
Currently, we have some css styles that override the theme styles. 
This PR is  moving some of them to dedicated files and adding a flag to optionally skip these styles via:

``` add_theme_support( 'sensei-learning-mode' );```


### Testing instructions
* Enable a theme
* Create a course
* Enable the learning mode
* Create a lesson
* Publish
* Start the course
* Go to the lesson (using the learning mode)
* Check the complete button
<img width="1071" alt="Screenshot 2022-11-03 at 18 47 29" src="https://user-images.githubusercontent.com/38718/199840802-04c3e746-1d9f-4a78-9476-59b0a2b63bf9.png">
* Enable the learning mode support via:


```
add_action( 'after_setup_theme', function() { 
    add_theme_support( 'sensei-learning-mode' );
} );
```

* Check again the lesson to see styles incoming from the theme. 
<img width="1170" alt="Screenshot 2022-11-03 at 18 32 45" src="https://user-images.githubusercontent.com/38718/199841226-c3e7f76a-81d0-4a6d-a79d-cd8208db29fa.png">
